src/pages/api/oauth/callback.ts

```diff
@@ -28,7 +28,7 @@ const handler: Handler = (request, response): Promise<void> => {
       `${request.query.code}`,
     );
 
-    span.setDisclosedAttribute("`authorization_code`", request.query.code);
+    span.setAttribute("`authorization_code`", request.query.code);
```

Comment:
Since we're here, let's add `span.setDisclosedAttribute("query.error", request.query.error);` :D